freecodecamp.conf.json

```diff
@@ -34,7 +35,7 @@
     },
     "landing": {
       "description": "Placeholder description",
-      "faq-link": "#",
+      "faq-link": "https://freecodecamp.org",
```

Comment:
This reminds me, we should probably come up with a better link for this. The announcement articles would probably work for now, we could maybe add a FAQ to those. I feel like we may want to link somewhere users can ask questions at some point - maybe discord. It's fine for now.

Reply:
I added this to test, but it is still not working because previews in VSCode automatically come with a `CORS same-origin` policy. So, far, I have not found a way to change that.